src/Servers/Kestrel/Core/src/Internal/Http/HttpParser.cs

```diff
@@ -384,36 +389,51 @@ private int ParseMultiSpanHeader(TRequestHandler handler, ref SequenceReader<byt
             header = currentSlice.Slice(reader.Position, lineEnd);
         }
 
-        var headerSpan = header.ToSpan();
+        byte[]? array = null;
+        int length = (int)header.Length;
```

Comment:
```suggestion
        var length = (int)header.Length;
```

Reply:
Just curious do we have coding guidelines for this repo? I couldn't find any. Perhaps we share https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/coding-style.md ? (Which would allow var here)

Reply:
Not written down in a markdown file (as far as I know).

It's the same but var wherever possible, no `s_` or `t_` prefixes, and file-scoped namespaces.

Reply:
We have https://github.com/dotnet/aspnetcore/wiki/Engineering-guidelines#coding-guidelines but it is likely outdated and not referenced anywhere.

Reply:
@danmoseley welcome to var, we use it everywhere 😄